### DISCUSSION TITLE
Source Links

### DISCUSSION THREAD

**Discussion by uo287841** [Arquisoft/wiq_en1a#59] [General]
Append links that can be usefull for the project.

**Comment by adriiglz** [comment]
[Check this website for orientation about WikiData access](https://www.markhneedham.com/blog/2020/01/29/newbie-guide-querying-wikidata/)

**Comment by ChristianFN2** [comment]
# Wikidata

- Wikidata API Authentication: https://www.wikidata.org/wiki/Wikidata:REST_API/Authentication

- Requests: https://doc.wikimedia.org/Wikibase/master/js/rest-api/

- Query builder: https://query.wikidata.org/querybuilder/?uselang=es

- Wikibase-sdk: https://github.com/maxlath/wikibase-sdk?tab=readme-ov-file
- JS library for Wikidata: https://www.npmjs.com/package/wikidatanpm

----------------

# React

- Introduction to React: https://es.react.dev/learn
- Tutorial 3 en linea: https://es.react.dev/learn/tutorial-tic-tac-toe

- Hooks: https://es.react.dev/reference/react/hooks

----------------

# MongoDB

- Manual: https://www.mongodb.com/docs/manual/
- Atlas SQL: https://www.mongodb.com/docs/atlas/data-federation/query/query-with-sql/

**Comment by adriiglz** [comment]
### Stacks
A stack is a group of development technologies that you can use together to create entire applications.
[These](https://codeless.co/web-development-stacks/) are some of the most popular full stacks for web development.

**Comment by uo287841** [comment]
### Connect React with MongoDB:
https://www.geeksforgeeks.org/how-to-connect-mongodb-with-reactjs/

**Comment by adriiglz** [comment]
moving this to a discussion
